signac/common/config.py

```diff
@@ -99,45 +98,33 @@ def read_config_file(filename, configspec=None, *args, **kwargs):
     return config
 
 
-def load_config(root=None, local=False):
-    """Load configuration, searching upward from a root path if desired.
+def load_config(root=None):
+    """Load configuration at root directory.
 
     Parameters
     ----------
     root : str
-        The path from which to begin searching for config files.
-    local : bool, optional
-        If ``True``, only search in the provided directory and do not traverse
-        upwards through the filesystem (Default value: False).
+        The path from which to load the local config.
```

Comment:
This sounds like it expects a config path, but I think it takes a project path. Also the use of the word "local" feels a bit out of place.

Reply:
See if the new wording is clearer to you.